components/PageFooter.vue

```diff
@@ -1,45 +1,105 @@
 <template>
   <footer
     data-qa="footer"
-    class="p-3"
+    class="py-5 px-3"
   >
     <b-container>
       <b-row>
         <b-col
-          cols="12"
-          lg="9"
+          lg="5"
+          class="pb-4"
         >
-          <ul
-            v-if="links"
-            class="footer-link-list m-0 p-0"
-          >
-            <li
-              v-for="(footerLink, index) in links"
-              :key="index"
+          <h5 class="text-uppercase font-weight-bold">
+            {{ $t('footer.ourMission') }}
+          </h5>
+          <p class="font-italic mb-0">
+            {{ $t('footer.ourMissionQuote') }}
+          </p>
+        </b-col>
+        <b-col
+          lg="4"
+          class="pb-4"
+        >
+          <b-row>
+            <b-col
+              col="6"
+              lg="12"
+            >
+              <h5 class="text-uppercase font-weight-bold">
```

Comment:
As we have two nearly identical sections of template here for the two link groups, make a new component, components/generic/LinkGroup.vue

Its top-level element should be `figure` and it should accept two props:
* `caption`: String, optional, rendered in `figcaption` if present
* `links`: Array, required, rendered in `ul > li`